TransportOperator/transportOperatorDT_class.f90

```diff
@@ -66,18 +66,24 @@ subroutine deltaTracking(self, p, tally, thisCycle, nextCycle)
       end if
 
       ! Check for void
-      if( p % matIdx() == VOID_MAT) cycle DTLoop
+      if(p % matIdx() == VOID_MAT) then
+        if (p % deltaFlight) call tally % reportInColl(p)
+        cycle DTLoop
+      end if
 
       ! Give error if the particle somehow ended in an undefined material
       if (p % matIdx() == UNDEF_MAT) then
         print *, p % rGlobal()
         call fatalError(Here, "Particle is in undefined material")
       end if
 
+      ! Accept both virtual and real collisions if virtualMode on
+      if (p % deltaFlight) exit DTLoop

```

Comment:
I don't think this is what we're going for. If you exit the loop, you'll go to collisionOperator and process the collision as if it was always real. I think you should check if the collision is real or virtual, and if virtual you call reportInColl. If real you exit the loop and go to collisionOperator as normal.

Reply:
Yes, my bad. Thanks for the feedback!